bootstrap.py

```diff
@@ -179,43 +177,8 @@ def remove_pyls_installation():
 
 atexit.register(remove_pyls_installation)
 
-print("*. Declaring completion providers")
-# Register completion providers
-fallback = pkg_resources.EntryPoint.parse(
-    'fallback = spyder.plugins.completion.providers.fallback.provider:'
-    'FallbackProvider')
-
-snippets = pkg_resources.EntryPoint.parse(
-    'snippets = spyder.plugins.completion.providers.snippets.provider:'
-    'SnippetsProvider'
-)
-
-lsp = pkg_resources.EntryPoint.parse(
-    'lsp = spyder.plugins.completion.providers.languageserver.provider:'
-    'LanguageServerProvider'
-)
-
-kite = pkg_resources.EntryPoint.parse(
-    'kite = spyder.plugins.completion.providers.kite.provider:'
-    'KiteProvider'
-)
-
-# Create a fake Spyder distribution
-d = pkg_resources.Distribution(__file__)
-
-# Add the providers to the fake EntryPoint
-d._ep_map = {
-    'spyder.completions': {
-        'fallback': fallback,
-        'snippets': snippets,
-        'lsp': lsp,
-        'kite': kite
-    }
-}
-
-# Add the fake distribution to the global working_set
-pkg_resources.working_set.add(d, 'spyder')
-
+subprocess.check_output([sys.executable, '-W', 'ignore',
+                         'setup.py', 'egg_info'])
```

Comment:
I was waiting to discuss this with @andfoy today. We decided we prefer to maintain the current way to handle completion entry points to avoid touching people's env.

We have put a lot of effort everywhere in the code to do this when Spyder is started with `bootstrap.py`, and this since the current approach is working, we don't see the need to remove it.

So please revert these changes and this PR will be ready.

Reply:
@mrclary, sorry but we need to merge your PR for our next alpha so I had to make these changes myself.